### DISCUSSION TITLE
Defaulting to Markdown

### DISCUSSION THREAD

**Discussion by bradcadams** [microsoft/azuredatastudio#14197] [Q&A]
When editing text cells, is there a way to change the default view to Markdown? Having to click Markdown every time I need to edit a text cell is not fun.

**Comment by Charles-Gagnon** [comment]
Currently you can only choose between the "Rich Text View" and the "Split View" using this setting : 

![image](https://user-images.githubusercontent.com/28519865/107259297-afa82800-69f1-11eb-9e5b-55a34a3279c3.png)

Are you saying that you would prefer to also be able to choose just straight Markdown as a 3rd option?

@chlafreniere @lucyzhang929 @barbaravaldez

**Reply by bradcadams** [reply]
Yes, I would like to be able to choose just straight Markdown as an option. The desired behavior would be when entered edit mode on a text cell that it open in Markdown rather than Rich Text.

![Azure_Data_Studio_Markdown](https://user-images.githubusercontent.com/59031174/107266650-6d8be000-6a0b-11eb-826e-8308d0558c92.gif)

**Reply by chlafreniere** [reply]
Reasonable request; we've heard this a couple of times over the past couple of weeks.  Tracking with #https://github.com/microsoft/azuredatastudio/issues/14204.
